spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -422,37 +444,41 @@ def main(argv: Sequence[str]):
                                                             group_funcs=(('SUM', func_sum),), length_pmj=length_from_pmj)
+
         else:
             # For other metrics, we compute the average and standard deviation across slices
             metrics_agg[key] = aggregate_per_slice_or_level(metrics[key], slices=slices,
                                                             levels=levels,
                                                             distance_pmj=distance_pmj, perslice=perslice,
                                                             perlevel=perlevel, fname_vert_level=fname_vert_level,
                                                             group_funcs=group_funcs, length_pmj=length_from_pmj)
+
     metrics_agg_merged = merge_dict(metrics_agg)
     # Normalize CSA values (MEAN(area))
     if arguments.normalize is not None:
-        data_subject = pd.DataFrame([arguments.normalize])
-        path_model = os.path.join(__sct_dir__, 'data', 'csa_normalization_models',
-                                  '_'.join(sorted(data_subject.columns)) + '.csv')
-        if not os.path.isfile(path_model):
-            parser.error('Invalid choice of predictors in -normalize. Please specify sex and brain-volume or sex, brain-volume and thalamus-volume.')
-        # Get normalization model
-        # Models are generated with https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/pipeline_ukbiobank/cli/compute_stats.py
-        # TODO update link with release tag.
-        data_predictors = pd.read_csv(path_model, index_col=0)
-        # Add interaction term
-        data_subject['inter-BV_sex'] = data_subject['brain-volume']*data_subject['sex']
-        for line in metrics_agg_merged.values():
-            line['MEAN(area)'] = normalize_csa(line['MEAN(area)'], data_predictors, data_subject)
+        if arguments.normalize != 'PAM50':
+            data_subject = pd.DataFrame([arguments.normalize])
+            path_model = os.path.join(__sct_dir__, 'data', 'csa_normalization_models',
+                                      '_'.join(sorted(data_subject.columns)) + '.csv')
+            if not os.path.isfile(path_model):
+                parser.error('Invalid choice of predictors in -normalize. Please specify sex and brain-volume or sex, brain-volume and thalamus-volume.')
+            # Get normalization model
+            # Models are generated with https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/pipeline_ukbiobank/cli/compute_stats.py
+            # TODO update link with release tag.
+            data_predictors = pd.read_csv(path_model, index_col=0)
+            # Add interaction term
+            data_subject['inter-BV_sex'] = data_subject['brain-volume']*data_subject['sex']
+            for line in metrics_agg_merged.values():
+                line['MEAN(area)'] = normalize_csa(line['MEAN(area)'], data_predictors, data_subject)
```

Comment:
Small nit: I think the `if arguments.normalize != 'PAM50':` can be removed too, and these changes can be undone (back to their original state).

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/f35f8a13d0aa8bb9096ca8af49b60337389f4531